pymysql/parsers.py

```diff
@@ -0,0 +1,20 @@
+from ._compat import PY2
+
+if PY2:
+    import ConfigParser as configparser
+else:
+    import configparser
+
+
+class DefaultFileParser(configparser.RawConfigParser):
+
```

Comment:
If this will be part of the 0.7 release, I can add this in now. Or we can wait and add it later. What's your thoughts? Add it now or wait?